python/cuml/preprocessing/LabelEncoder.py

```diff
@@ -30,6 +29,12 @@ def _enforce_str(y: cudf.Series) -> cudf.Series:
     return y
 
 
+def _check_npint32(y: cudf.Series) -> cudf.Series:
```

Comment:
This feels misnamed. It's doing more than checking if y is np.int32 dtype. It feels like it's enforcing that y be np.int32

Reply:
Oh yeah, how about _ensure_npint32() or _convert_to_npint32()?

Reply:
renamed to _enforce_npint32()